docs/api.rst

```diff
@@ -174,6 +174,8 @@ Writes the given object to the graph, connected to the given parent.
   profile is the parent of a feed, and a post is the parent of a comment.
 * ``connection_name`` - A ``string`` that specifies the connection or edge
   between objects, e.g., feed, friends, groups, likes, posts.
+* ``file`` - A file object representing like the image to be uploaded.
```

Comment:
I think "representing an image to be uploaded" would make this sentence flow better.